src/main/java/redis/clients/jedis/commands/SetCommands.java

```diff
@@ -42,6 +42,10 @@
 
   long sinterstore(String dstkey, String... keys);
 
+  long sintercard(String... keys);
```

Comment:
We'd better add Javadoc when adding new interfaces

Reply:
I added Javadoc in Jedis.py, and non of the commands here are documented 
Do you want me to add it here also?

Reply:
@AvitalFineRedis It's better to add javadoc in interfaces. Then you may or may not add javadoc in classes.